api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -285,18 +280,14 @@ func (c *imageBuilder) createKanikoJobSpec(project mlp.Project, model *models.Mo
 	kanikoPodName := c.nameGenerator.generateBuilderJobName(project, model, version)
 	imageRef := c.imageRef(project, model, version)
 
-	var labels = map[string]string{
+	labels := map[string]string{
 		labelTeamName:         project.Team,
 		labelStreamName:       project.Stream,
 		labelAppName:          model.Name,
 		labelEnvironment:      c.config.Environment,
 		labelOrchestratorName: "merlin",
 	}
 
-	for _, label := range project.Labels {
```

Comment:
Please help me refresh my memory, why we remove this?

Reply:
because previously it is breaking the label name constraints that used in k8s, since it is not used for cost labelling i prefer to remove it